src/crypto/tls/handshake_client.go

```diff
@@ -122,7 +122,7 @@ func (c *Conn) makeClientHello(minVersion uint16) (*clientHelloMsg, ecdheParamet
 		hello.supportedSignatureAlgorithms = config.supportedSignatureAlgorithms()
 	}
 
-	var params ecdheParameters
+	var secret clientKeySharePrivate
```

Comment:
Since we're (unavoidably) overloading the semantics of `curveID` to include KEMs, perhaps we could keep this name as well for consistency.
```suggestion
	var params clientKeySharePrivate
```

Reply:
I prefer to keep the change as it's consistent with the patch that we maintain in go-boring-pq.